### DISCUSSION TITLE
How can I use multiple monitors with the same description?

### DISCUSSION THREAD

**Discussion by Jax-Tsai-zero** [hyprwm/Hyprland#6178] [Q&A]
I have two monitors with the same name and description, how can I distinguish them in hyprland?
(Port name seems not that reliable)

**Comment by MightyPlaza** [comment]
do they have the same serial number as well?

**Reply by Jax-Tsai-zero** [reply]
Actually some monitors that do not meet specifications do not have serial numbers.🥲
